src/components/vlplotgroup/vlplotgroup.js

```diff
@@ -153,6 +153,17 @@ angular.module('vlui')
           return false;
         };
 
+        scope.select = function(chart) {
+          Logger.logInteraction(Logger.actions.SPEC_SELECT, chart.shorthand, {
+            list: scope.listTitle
+          });
+          Pills.parse(chart.vlSpec);
+          if (scope.postSelectAction) {
```

Comment:
Where is this from?   

We don't have `scope.postSelectAction` here.  

Thus, the main view won't automatically scroll to the top when you selects a plot in the main view. 
(But we should make it scroll to the top if we select a view in the bookmark view too.)